src/metrics/mod.rs

```diff
@@ -38,6 +42,29 @@ pub static LRU_CACHE_MISS: Lazy<Family<KindLabel, Counter>> = Lazy::new(|| {
     metric
 });
 
+pub static RPC_METHOD_FAILURE: Lazy<Family<RpcMethodLabel, Counter>> = Lazy::new(|| {
+    let metric = Family::default();
+    DEFAULT_REGISTRY.write().register(
+        "rpc_method_failure",
+        "Number of failed RPC calls",
+        metric.clone(),
+    );
+    metric
+});
+
+pub static RPC_METHOD_TIME: Lazy<Family<RpcMethodLabel, Histogram>> = Lazy::new(|| {
+    let metric = Family::<RpcMethodLabel, Histogram>::new_with_constructor(|| {
+        // Histogram with 10 buckets starting from 0.01s going to 5.12s, each bucket twice as big as the last.
+        Histogram::new(exponential_buckets(0.01, 2.0, 10))
+    });
+    crate::metrics::default_registry().register(
```

Comment:
Hi, I'm curious. Is there a particular reason we do not use `DEFAULT_REGISTRY.write()` here?

Reply:
the `default_registry()` hides the detail that the registry is under an `RwLock`.

Reply:
But, I see that all other metrics use `DEFAULT_REGISTRY.write()` in this file.

Reply:
They likely shouldn't. :)